src/App.scss

```diff
@@ -21,3 +21,32 @@
 .message-body {
   white-space: pre-line;
 }
+
+@media print, screen and (min-width: 769px) {
```

Comment:
The media query syntax `print, screen and (min-width: 769px)` might not work as intended. Consider using separate media queries or ensuring that the conditions are correctly combined for your use case.